common/list/list-group-en.html

```diff
@@ -0,0 +1,167 @@
+---
+{
+	"title": "List Group",
```

Comment:
Use sentence case as required per our Canada.ca style guide - https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-1-1

```suggestion
	"title": "List group",
```